README.md

```diff
@@ -1435,7 +1435,7 @@ $ trivy --reset
 
 - [Remic](https://github.com/aquasecurity/remic)
   - Vulnerability Scanner for Detecting Publicly Disclosed Vulnerabilities in Application Dependencies
-
+  
```

Comment:
Is this necessary?

Reply:
@knqyf263 No, It is added after resolving conflicts. I have removed the extra line.